tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -125,6 +126,38 @@ func New(db *pgxpool.Pool) Store {
     }
 }
 
+//// Used for testing
```

Comment:
nit:  I think the testing functions should remain at the bottom though that may just be my personal preference.